Packs/PAN-OS/Integrations/Panorama/Panorama.yml

```
@@ -5997,7 +5997,7 @@ script:
       - corr
       - system
       - decryption
-    - description: The query string by which to match criteria for the logs. This is similar to the query provided in the web interface under the Monitor tab when viewing the logs.
+    - description: The query string by which to match criteria for the logs. This is similar to the query provided in the web interface under the Monitor tab when viewing the logs. You can use the query argument for a custom query or use a set of the pre-defiend command arguemnts (but not both) - time-generated, time-generated-after, addr-src, addr-dst, zone-src, zone-dst, action, port-dst, rule, url, filedigest.
```

```suggestion
    - description: "The query string by which to match criteria for the logs. This is similar to the query provided in the web interface under the Monitor tab when viewing the logs. You can use the query argument for a custom query or use a set of the pre-defined command arguments (but not both): time-generated, time-generated-after, addr-src, addr-dst, zone-src, zone-dst, action, port-dst, rule, url, filedigest."
```
(Added quotes for the yml formatting)